ParkingSlotsTest/Controllers/ParkingSlotControllerTests.cs

```diff
@@ -53,5 +55,100 @@ public void GivenNothing_WhenIndexIsCalled_ThenReturnsViewResult()
             Assert.Equal(JsonSerializer.Serialize(expectedVMs), JsonSerializer.Serialize(model));
             Assert.NotNull(result);
         }
+        #endregion
+
+        #region Create
+        [Fact]
+        public void GivenParkingZoneId_WhenCreateIsCalled_ThenReturnsViewResult()
+        {
+            //Arrange
+            
+            //Act
+            var result = _controller.Create(Id);
+
+            //Assert
+            Assert.NotNull(result);
+            var model = Assert.IsType<ViewResult>(result).Model;
+            Assert.IsType<CreateVM>(model);
```

Comment:
this action returns CreateVM so that check this using Serializer whats returning and whats expecting